samples/dnn/text_detection.py

```diff
@@ -0,0 +1,146 @@
+# Import required modules
+import cv2 as cv
+import math
+import argparse
+
+############ Add argument parser for command line arguments ############
+parser = argparse.ArgumentParser(description='Use this script to run text detection deep learning networks using OpenCV.')
```

Comment:
It'd be nice to add some references to origin paper and network's repo. I.e. like in text_detection.cpp:

```
Use this script to run TensorFlow implementation (https://github.com/argman/EAST) of
EAST: An Efficient and Accurate Scene Text Detector (https://arxiv.org/abs/1704.03155v2)
```